packages/http/src/models/Sum.ts

```diff
@@ -21,30 +22,65 @@ function floatToBalance(floatValue: number): Balance<NetworkTokens> {
 export default class Sum extends DataModel {
   private client: Client
 
-  public total: Balance<NetworkTokens>
+  public total: number
 
-  public stddev: Balance<NetworkTokens>
+  public balanceTotal: Balance<NetworkTokens>
 
-  public min: Balance<NetworkTokens>
+  public stddev: number
 
-  public median: Balance<NetworkTokens>
+  public balanceStddev: Balance<NetworkTokens>
 
-  public max: Balance<NetworkTokens>
+  public min: number
 
-  public avg: Balance<NetworkTokens>
+  public balanceMin: Balance<NetworkTokens>
 
-  constructor(client: Client, rewards: HTTPSum) {
+  public median: number
+
+  public balanceMedian: Balance<NetworkTokens>
```

Comment:
maybe rather than public members that get populated during the constructor, they could be getters that are only constructed if called?

```ts
get balanceTotal() {
  return floatToBalance(this.total)
}
```

Reply:
by making it a `get` it's called the same way: `sum.balanceTotal` but it's just computed at the time it's called

Reply:
How do those TS getters work? Because I think it defines it as a property on the object anyway? Or do you call them as a function?

Reply:
Ah ok, let me update them

Reply:
you don't have to call them as a function, they are called like properties, but I believe they're computed at the time they're called. So you wouldn't have to do `balanceTotal()` you could just call `.balanceTotal` but you wouldn't have to compute balances for things where it doesn't make sense, like witnesses